src/parser/abstractparser.cpp

```diff
@@ -1201,6 +1285,7 @@ void AbstractParser::offlineCharacterMove(CommandIdType direction)
     }
   }
   m_mapData->unselect(rs1);
+  usleep(40000);
```

Comment:
This is a hackish fix to the following issue: in trunk, speedwalking doesn't work offline.  This makes it work (at least for me, at my computer's speed).

I don't really understand the root of the issue.  This makes it a little bit slow for long journeys, but at the same time it lets you watch the path taken, so it's not too bad.